src/Authentication.Abstractions/Properties/AssemblyInfo.cs

```diff
@@ -43,5 +43,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.0.0.1")]
```

Comment:
This sill likely cause a problem the next time we release this library for Az - we will likely need to put this back to avoid any backwards compatibility issues.

Reply:
The latest common AssemblyVersion is 1.0.0.0. No idea why it was changed to 1.0.0.1 after Azure was released.